catroid/src/main/java/org/catrobat/catroid/io/SoundManager.java

```diff
@@ -51,11 +54,23 @@ public static SoundManager getInstance() {
 	}
 
 	public synchronized void playSoundFile(String pathToSoundfile) {
+		playSound(pathToSoundfile, 0);
+	}
+
+	public synchronized void playSoundFileWithStartTime(String pathToSoundfile,
```

Comment:
The name is ok, but it is the only difference to playSound. Please remove it and/or rename Play Sound)